src/views/Layout.jsx

```diff
@@ -24,17 +24,17 @@ export function Layout() {
 					<Outlet />
 				</main>
 				<nav className="Nav">
-					{!!user ? <SignOutButton /> : <SignInButton />}
 					<div className="Nav-container">
-						<button href="#" className="Nav-link">
+						{!!user ? <SignOutButton /> : <SignInButton />}
+						<NavLink to="/" className="Nav-link">
```

Comment:
This looks great! One small suggestion here would be to consider adding ARIA attributes to the interactive navigation links to improve accessibility. For instance:
```
<NavLink to="/" className="Nav-link" aria-label="Home">
  Home
</NavLink>
```

Reply:
Good point! Definetly can add all the aria-label attributes to the links. Any others that you think should be added?

Reply:
Today I learned about Aria labels! A couple weeks ago, I was just thinking "When I use React with React Router, I wonder what other accessibility features are available for screen readers????"